### DISCUSSION TITLE
Is there any way to earn money by making apps

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#153] [General]
Like putting ads on the app . Or just selling the full versions. Or some thing like that

**Comment by joaquinperaza** [comment]
I think Amazfit should give some rewards at least some great discounts for devs. I have a nice set of instruments for aviation already coded but won't release for free.

**Comment by zdunex25** [comment]
You guys can sell apps and watchfcaes in zepp store, you just need a confirmed by personal id (and few other steps) zepp developer account.
